client/keys/migrate_test.go

```diff
@@ -27,23 +27,21 @@ type setter interface {
 type MigrateTestSuite struct {
 	suite.Suite
 
-	dir  string
-	n1   string
-	cdc codec.Codec
-	priv cryptotypes.PrivKey
-	pub  cryptotypes.PubKey
-
+	dir     string
+	appName string
+	cdc     codec.Codec
+	priv    cryptotypes.PrivKey
+	pub     cryptotypes.PubKey
 }
 
 func (s *MigrateTestSuite) SetupSuite() {
 	s.dir = s.T().TempDir()
 	s.cdc = simapp.MakeTestEncodingConfig().Codec
-	s.n1 = "cosmos"
+	s.appName = "cosmos"
```

Comment:
@robert-zaremba Following your suggestion, I renamed `s.n1` to `s.appName`. Does it sound good to you?